spec/struct.dd

```diff
@@ -207,6 +221,8 @@ $(H2 $(LNAME2 default_union_init, Default Initialization of Unions))
         ---
         )
 
+        $(P It is an error to supply initializers for members other than the first one.)
```

Comment:
This might change once we have named parameters, though the DIP doesn't mention unions.